*** cygwinccompiler.py	Tue Mar  8 09:43:14 2011
--- cygwinccompiler_patched.py	Mon Apr  8 23:20:14 2013
***************
*** 319,329 ****
          else:
              entry_point = ''
  
!         self.set_executables(compiler='gcc -mno-cygwin -O -Wall',
!                              compiler_so='gcc -mno-cygwin -mdll -O -Wall',
!                              compiler_cxx='g++ -mno-cygwin -O -Wall',
!                              linker_exe='gcc -mno-cygwin',
!                              linker_so='%s -mno-cygwin %s %s'
                                          % (self.linker_dll, shared_option,
                                             entry_point))
          # Maybe we should also append -mthreads, but then the finished
--- 319,329 ----
          else:
              entry_point = ''
  
!         self.set_executables(compiler='gcc  -O -Wall',
!                              compiler_so='gcc  -mdll -O -Wall',
!                              compiler_cxx='g++  -O -Wall',
!                              linker_exe='gcc ',
!                              linker_so='%s  %s %s'
                                          % (self.linker_dll, shared_option,
                                             entry_point))
          # Maybe we should also append -mthreads, but then the finished
